doc/build/build.md

```diff
@@ -289,31 +320,34 @@ Another useful option is `--output-on-failure`, which will print verbose output
 in case a test fails. Full set of options can be found [here](https://cmake.org/cmake/help/latest/manual/ctest.1.html).
 
 @warning
-When using the `/opt/intel/oneapi/setvars.sh` script from the Intel oneAPI Toolkit,
-`LD_LIBRARY_PATH` is set to include the oneDNN library path from the installation.
-Make sure the correct oneDNN library is present in
-`LD_LIBRARY_PATH` by setting it explicitly if needed.
+When using the `/opt/intel/oneapi/setvars.sh` script from the Intel oneAPI toolkit,
```

Comment:
IPLDT (linter) is currently unhappy about Toolkit -> toolkit change, if this one is indeed correct, I could update the hits.db after this PR is merged
@ranukund please let me know